src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeStreamInfoItemExtractor.java

```diff
@@ -94,13 +94,23 @@ public String getUploaderName() throws ParsingException {
         }
     }
 
+    private static String fixRelativeUrls(String url) {
+        if(url == null || url.isEmpty()) return url;
+        if(url.startsWith("/")) {
+            return "https://youtube.com" + url;
+        } else {
+            return url;
+        }
+    }
+
     @Override
     public String getUploaderUrl() throws ParsingException {
         try {
             try {
-                return item.select("div[class=\"yt-lockup-byline\"]").first()
+                String url = item.select("div[class=\"yt-lockup-byline\"]").first()
                         .select("a").first()
                         .attr("href");
+                return fixRelativeUrls(url);
```

Comment:
Instead of checking manually, just use `abs:<attr>` like this: [YoutubeStreamInfoItemExtractor.java#L55](https://github.com/TeamNewPipe/NewPipeExtractor/blob/master/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeStreamInfoItemExtractor.java#L55)

In fact, there's some more places where this happens, you can fix them in this pull request if you want.

Reply:
Good to know. Fixed